torchaudio/csrc/ffmpeg/stream_reader/stream_reader.h

```diff
@@ -52,32 +65,118 @@ class StreamReader {
   // Query methods
   //////////////////////////////////////////////////////////////////////////////
  public:
-  // Find a suitable audio/video streams using heuristics from ffmpeg
+  /// @name Query methods
+  ///@{
+
+  /// Find a suitable audio stream using heuristics from ffmpeg.
+  ///
+  /// If successful, the index of the best stream (0>=) is returned.
+  /// otherwise a negative value is returned.
   int64_t find_best_audio_stream() const;
+  /// Find a suitable video stream using heuristics from ffmpeg.
+  ///
+  /// If successful, the index of the best stream (0>=) is returned.
+  /// otherwise a negative value is returned.
   int64_t find_best_video_stream() const;
-  // Fetch metadata of the source
+  /// Fetch metadata of the source media.
   OptionDict get_metadata() const;
-  // Fetch information about source streams
+  /// Fetch the number of source streames found in the input media.
+  ///
+  /// The source streams include not only audio/video streams but also
+  /// subtitle and others.
   int64_t num_src_streams() const;
+  /// Fetch information about the specified source stream.
+  ///
+  /// The valid value are ``[0, num_src_streams())``.
   SrcStreamInfo get_src_stream_info(int i) const;
-  // Fetch information about output streams
+  /// Fetch the number of output streams defined by client code.
+  ///
+  /// @addtogroup stream_reader_query_methods
   int64_t num_out_streams() const;
+  /// Fetch information about the specified output stream.
+  ///
+  /// The valid value are ``[0, num_out_streams())``.
   OutputStreamInfo get_out_stream_info(int i) const;
-  // Check if all the buffers of the output streams are ready.
+  /// Check if all the buffers of the output streams have enough decoded frames.
   bool is_buffer_ready() const;
 
+  ///@}
+
   //////////////////////////////////////////////////////////////////////////////
   // Configure methods
   //////////////////////////////////////////////////////////////////////////////
-  void seek(double timestamp_s, int64_t mode);
+  /// @name Configure methods
+  ///@{
 
+  /// Define an output audio stream.
+  ///
+  /// @param i The index of the source stream.
+  ///
+  /// @param frames_per_chunk Number of frames returned as one chunk.
+  /// @parblock
+  ///   If the source stream is exhausted before enough frames are buffered,
+  ///   then the chunk is returned as-is.The number of frames in one chunk.
```

Comment:
is the second sentence not finished?
```suggestion
  ///   then the chunk is returned as-is. The number of frames in one chunk.
```